### ISSUE TITLE
5.0: Cancel Sync from Conflict Resolution does not cancel sync

### ISSUE THREAD

**Issue by fcheslack** [zotero/zotero#969]
Closes dialog, but sync continues.

I suspect there were some other problems with the way conflict resolution was behaving with multiple file modification conflicts, but I'm not entirely sure what I should expect from it with API syncing.

**Comment by dstillman** [comment]
Yeah, so as you noted on IRC, it's actually going on to the next library. Is that what we want?

**Comment by fcheslack** [comment]
I'd say that, to an end user, sync looks like a single operation, so it should be treated as such in this case (when the user is taking explicit action on it).

**Comment by dstillman** [comment]
New plan: "Cancel Sync", "Skip This Library", and "Next/Finish"
